cypress/support/step_definitions/validations.md

```diff
@@ -445,6 +445,88 @@ While validating, if a key is present in both fcs-validation jsons (eg: cypress/
         ]
 }
 
+# Dynamic Content Validation
+
+### Background:
+For the fixture validations mentioned in [Supported Validations](./validations.md#supported-validations), deviceContentValidation is used when the validation is done based on device specific value.
+For eg :
+
+```
+    "DEVICE_ID": {
+        "data": [
+            {
+                "type": "fixture",
+                "validations": [
+                    {
+                        "mode": "deviceContentValidation",
+                        "type": "DEVICEID",
+                        "description": "Validation of the Device Id Format"
+                    }
+                ]
+            }
+        ]
+    }
+
+```
+
+When the validation mode is "deviceContentValidation", the source of truth for the validation can be taken in two ways:
+1. Static data 
+2. Dynamic data
+
+Static data :
+For static data, the source of truth is stored in a <deviceMac.json> file placed [here](../../fixtures/external/devices/) based on the deviceMac value being tested. When the test begins, these values are loaded into an object stored in an environment variable called deviceData. If deviceMac.json is not found in the specified path, the values are saved as they are, for example: DEVICEID: "DEVICEID".
```

Comment:
[here](../../fixtures/external/devices/)  link doesnt work. How will it help a user to be provided with this link ? 
 FCS does not have a external folder by default . External folder comes whenever fcs is installed with a config module. Provide the actual path of file to be added in config module repo

Reply:
added a generic path which is common for all configModule